project/AkkaBuild.scala

```diff
@@ -119,6 +119,8 @@ object AkkaBuild extends Build {
       persistence, distributedData,
       testkit
     ).map(_ % "compile;compile->test;provided->provided")
+  ).settings(
+    sources in doc in Compile := List()
```

Comment:
please remove here and in stead in `akka-bench-jmh/build.sbt` add this line: `AkkaBuild.dontPublishSettings`